scipy/signal/signaltools.py

```diff
@@ -899,7 +899,14 @@ def medfilt(volume, kernel_size=None):
     out : ndarray
         An array the same size as input containing the median filtered
         result.
-
+        
+    See also
+    --------
+    ndimage.filters.median_filter
+    
+    Notes
+    -------
+    The more general function `ndimage.filters.median_filter` has a more efficient implementation of a median filter and therefore runs much faster.
```

Comment:
FYI for new code at least we try to satisfy the PEP8 line-length convention

Reply:
thanks for fixing it! I always forget that on my extra wide screen...